code/modules/mob/living/silicon/robot/drone/drone_console.dm

```diff
@@ -6,122 +6,132 @@
 	req_access = list(ACCESS_ENGINE_EQUIP)
 	circuit = /obj/item/circuitboard/drone_control
 
-	//Used when pinging drones.
-	var/drone_call_area = "Engineering"
-	//Used to enable or disable drone fabrication.
+	/// The linked fabricator
 	var/obj/machinery/drone_fabricator/dronefab
+	/// Used when pinging drones
+	var/drone_call_area = "Engineering"
+	/// Cooldown for area pings
+	var/ping_cooldown = 0
 
-/obj/machinery/computer/drone_control/attack_ai(var/mob/user as mob)
-	return src.attack_hand(user)
-
+/obj/machinery/computer/drone_control/Initialize(mapload)
+	..()
```

Comment:
I wasn't sure if `. = ..()` would maybe cause the console to initialize before the fabricator here, since it probably wouldn't connect at roundstart then.

Reply:
That would be pretty wild if it did do something like that. All this change *should* do is make it so the Initialize returns the parent call value, and not null.

Reply:
Definitely missed something about the use of `..()` then, ah well.